docker/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.2'
```

Comment:
Why the change @JoaoCampos89 ? 
@clemsos do you think it is ok? We should do it in a separate PR!

Reply:
I needed to do this to run locally on my machine. I can revert this.